kano_world/functions.py

```diff
@@ -223,10 +213,7 @@ def get_mixed_username():
 
 
 def get_token():
-    try:
-        return load_profile()['token']
-    except Exception:
-        return ''
+    return has_token()
```

Comment:
This implementation should be what you already had for `has_token()` and the `has_token()` simply returns a boolean.

Reply:
same as above, also tested